### ISSUE TITLE
[Bug] Automap marker not wiped after map change

### ISSUE THREAD

**Issue by NightFright2k19** [JNechaevsky/international-doom#89]
At least in Inter-Doom, placed map markers are persistent between level changes instead of being cleared.

**Comment by JNechaevsky** [comment]
Oh, my third favorite thing to break! First one is RAW screens, second one is ENDOOM screens. 😀

Thank you, should be fine now. Heretic and Hexen seems to be fine.

**Comment by NightFright2k19** [comment]
Automap oopsies are your favorite thing, admit it...! xD

**Comment by JNechaevsky** [comment]
Yeah, just fixed it. This only confirms that making things in a hurry is a bad practice. I was about to run away from day job, and here's the result. 😐
